Update issue templates (#4728)
ktorio/ktor: .github/ISSUE_TEMPLATE/issue_report.md

---
name: Bug report or Feature request
about: Create an issue to help us improve
title: ''
labels: ''
assignees: ''

---

> 🚨 Please Note: We manage all our issues and tickets through [YouTrack](https://youtrack.jetbrains.com/issues?q=project:%20KTOR) for better tracking and faster feedback.

Reviewer: ```suggestion
🚨 Important: We manage all our issues and tickets in [YouTrack](https://youtrack.jetbrains.com/issues?q=project:%20KTOR) for better tracking and faster feedback.
```

> 📌 If you have a bug report, feature request, or any other issue, kindly file it in our [YouTrack project](https://youtrack.jetbrains.com/newissue?project=ktor).

Reviewer: ```suggestion
📌 To report a bug, request a feature, or submit an issue, please use our [YouTrack project](https://youtrack.jetbrains.com/newissue?project=ktor).
```
